scripts/simplify_network.py

```diff
@@ -620,6 +766,20 @@ def consense(x):
 
     update_p_nom_max(n)
 
+    n = remove_empty_isolated_nodes(n)
+
+    e_threshold_drop_isolated = cluster_config.get("e_threshold_drop_isolated", False)
+    e_threshold_merge_isolated = cluster_config.get("e_threshold_merge_isolated", False)
+    if e_threshold_drop_isolated:
+        n = drop_isolated_nodes(n, drop_e_threshold=e_threshold_drop_isolated)
+    if e_threshold_merge_isolated:
+        n, merged_nodes_map = merge_isolated_nodes(
+            n,
+            merge_e_threshold=e_threshold_merge_isolated,
+            aggregation_strategies=aggregation_strategies,
+        )
+        busmaps.append(merged_nodes_map)
```

Comment:
do we store busmaps somewhere?
Should we drop this line?

Reply:
As far as I understand, all the busmaps are being collected into the `busmap` list to be reduced and written into a file

Reply:
But maybe before than here?
I don't see any saving afterwards

Reply:
I have tried to keep close to the approach used by `aggregate_to_substations( )`. We have `cluster_regions( )` in the very end:

https://github.com/ekatef/pypsa-earth/blob/b0f3f8a476a71a6b660406d987aad254030f2c96/scripts/simplify_network.py#L788

which is defined in `cluster_networks` and contains output to the file

https://github.com/ekatef/pypsa-earth/blob/b0f3f8a476a71a6b660406d987aad254030f2c96/scripts/cluster_network.py#L607-L618

Reply:
Ok! Though my feeling is that the new busmap may not be compatible with the old one.
Maybe we should pipe them, otherwise we lose retrocompatibility.
if the first busmap maps 1 into 2 and then the second one 2 into 3.
Probably, the expected busmap to save is 1->3.
This enables compatibility with the current formulation I think